src/sentry/static/sentry/app/views/performance/traceDetails/content.tsx

```diff
@@ -15,6 +24,69 @@ type Props = {
 };
 
 class TraceDetailsContent extends React.Component<Props> {
+  renderTraceLoading() {
+    return <LoadingIndicator />;
+  }
+
+  renderTraceNotFound() {
+    return <LoadingError message={t('The trace you are looking for was not found.')} />;
+  }
+
+  renderTrace(trace) {
+    const traceInfo = getTraceInfo(trace);
+
+    return (
+      <TraceDetailHeader>
+        <MetaData
+          headingText={t('Transactions')}
+          tooltipText={t('All the transactions that are a part of this trace.')}
+          bodyText={t(
+            '%s of %s',
+            traceInfo.relevantTransactions,
+            traceInfo.totalTransactions
+          )}
+          subtext={tn(
+            'Across %s project',
+            'Across %s projects',
+            traceInfo.relevantProjects
+          )}
+        />
+      </TraceDetailHeader>
+    );
+  }
+
+  renderContent() {
+    const {location, organization, traceSlug} = this.props;
+    const {query} = location;
+    const start = decodeScalar(query.start);
+    const end = decodeScalar(query.end);
+
+    if (!start || !end) {
+      // throw new Error('No date range selection found.');
+      return this.renderTraceNotFound();
```

Comment:
Can we just have an explicit Error message here "Trace view requires both start & end" since the only way for a user to see this should be to edit the URL? That way since they're a user that's comfortable editing the URL, they know to set the start&end params as well

Reply:
I can do that for now, and we can refine the error message more later on.

Also going to start tracking this, forgot to send it to Sentry after I commented it out temporarily. This will help in finding bugs like which links forgot to set a date range since we should be able to see it in the breadcrumbs.